ui/app/AppLayouts/Wallet/panels/ManageCollectiblesPanel.qml

```diff
@@ -0,0 +1,239 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ 0.1
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Models 0.1
+
+import utils 1.0
+import shared.controls 1.0
+
+import AppLayouts.Wallet.controls 1.0
+
+Control {
+    id: root
+
+    required property var baseModel
+
+    readonly property bool dirty: d.controller.dirty
+
+    background: null
+
+    function saveSettings() {
+        d.controller.saveSettings();
+    }
+
+    function revert() {
+        d.controller.revert();
+    }
+
+    function clearSettings() {
+        d.controller.clearSettings();
+    }
+
+    QtObject {
+        id: d
+
+        property bool collectionGroupsExpanded: true
+        property bool communityGroupsExpanded: true
+
+        readonly property var renamedModel: RolesRenamingModel {
+            sourceModel: root.baseModel
+
+            mapping: [
+                RoleRename {
```

Comment:
I'm not sure about that.. Isn't the base model going to have a role called  `symbol `yet? Is / should this `uid` be the `key` role to uniqually identify the item?

Reply:
Ok so.. `uid` contains information about the `symbol`, so we should also keep the `key` and the `symbol`. The `key`(although in the major cases will be == to`symbol` value) will uniqualy identify the element as said before!